tests/zfs-tests/include/libtest.shlib

```diff
@@ -3416,3 +3416,49 @@ function swap_cleanup
 
 	return 0
 }
+
+#
+# Set a global system tunable (64-bit value)
+#
+# $1 tunable name
+# $2 tunable values
+#
+function set_tunable64
+{
+	set_tunable_impl "$1" "$2" Z
+}
+
+#
+# Set a global system tunable (32-bit value)
+#
+# $1 tunable name
+# $2 tunable values
+#
+function set_tunable32
+{
+	set_tunable_impl "$1" "$2" W
+}
+
+function set_tunable_impl
```

Comment:
This is a good idea.  I wonder if we should have get_tunable analogs, which can be used to capture the default value before a test and restore it after the test is done.